pennylane/_grad.py

```diff
@@ -15,19 +15,39 @@
 This module contains the autograd wrappers :class:`grad` and :func:`jacobian`
 """
 import warnings
+from functools import partial, wraps
 
 from autograd import jacobian as _jacobian
 from autograd.core import make_vjp as _make_vjp
 from autograd.extend import vspace
 from autograd.numpy.numpy_boxes import ArrayBox
 from autograd.wrap_util import unary_to_nary
 
+from pennylane.capture import create_grad_primitive, enabled
 from pennylane.compiler import compiler
 from pennylane.compiler.compiler import CompileError
 
 make_vjp = unary_to_nary(_make_vjp)
 
 
+def _capture_diff(func, argnum=None, diff_prim=None):
```

Comment:
This will be used for `qml.grad` and `qml.jacobian`. The kwarg `diff_prim` specifies which one it is.